Experiments section (missing MAD outlier removal experiment) (#54)
chhoumann/thesis-chemcam: report_pre_thesis/src/sections/experiments/ica_outlier.tex

\subsection{Experiment: ICA MAD Outlier Removal}\label{sec:experiment_ica_mad_outlier_removal}
In the ICA phase, the original authors employed the Median Absolute Deviation (MAD) for outlier removal, yet the detailed methodology of their approach was not fully delineated.
Consequently, in our version of the pipeline, we chose to exclude the outlier removal step during the ICA phase to avoid introducing unsubstantiated assumptions, as described in Section~\ref{sec:ica_data_preprocessing}.

> This decision allows us to evaluate the intrinsic effectiveness of the ICA phase without the influence of outlier removal.

Reviewer: ```suggestion
This decision allowed us to evaluate the intrinsic effectiveness of the ICA phase without the influence of outlier removal.
```

> Introducing outlier removal using MAD in our replication of the pipeline presents an opportunity to assess its impact on the pipeline's efficacy.

Reviewer: ```suggestion
Introducing outlier removal using MAD in our replication of the pipeline presented an opportunity to assess its impact on the pipeline's efficacy.
```

> This decision allows us to evaluate the intrinsic effectiveness of the ICA phase without the influence of outlier removal.
> Introducing outlier removal using MAD in our replication of the pipeline presents an opportunity to assess its impact on the pipeline's efficacy.
> By comparing the results with and without MAD, we can quantitatively measure the utility of this step.
> Such an experiment is crucial for understanding whether MAD significantly contributes to reducing noise and improving data quality, thereby enhancing the overall performance of the machine learning pipeline.
> This experiment could also offer insights into the robustness of the ICA phase against outliers, providing a more comprehensive understanding of the pipeline's capabilities and limitations.

Reviewer: I feel this could be written more consicely. Here is a suggestion:
```suggestion
This decision allows us to evaluate the intrinsic effectiveness of the ICA phase without outlier removal and assesses the impact of introducing MAD (Median Absolute Deviation) for outlier removal in our pipeline replication. 
By comparing results with and without MAD, we aim to quantitatively determine its utility in reducing noise and improving data quality. 
This will also provide insights into the robustness of the ICA phase against outliers, offering a comprehensive understanding of the pipeline's capabilities and limitations.
```
